api/filters.go

```diff
@@ -286,46 +332,63 @@ func (api *FilterAPI) getFilterBlueprintDimensionOptions(w http.ResponseWriter,
 	w.WriteHeader(http.StatusOK)
 	_, err = w.Write(bytes)
 	if err != nil {
-		log.Error(err, log.Data{"filter_blueprint_id": filterID, "dimension_options": dimensionOptions})
+		log.Error(err, logData)
 		setErrorCode(w, err)
 		return
 	}
 
-	log.Info("got filtered blueprint", log.Data{"filter_blueprint_id": filterID, "dimension_options": dimensionOptions})
+	log.Info("got filtered blueprint", logData)
 }
 
 func (api *FilterAPI) getFilterBlueprintDimensionOption(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	filterID := vars["filter_blueprint_id"]
 	name := vars["name"]
 	option := vars["option"]
+	logData := log.Data{
+		"func":                "getFilterBlueprintDimensionOption",
+		"filter_blueprint_id": filterID,
+		"dimension":           name,
+		"option":              option,
+	}
+	log.Info("get filter blueprint dimension option", logData)
 
 	err := api.dataStore.GetFilterDimensionOption(filterID, name, option)
 	if err != nil {
-		log.Error(err, log.Data{"filter_blueprint_id": filterID, "dimension": name, "option": option})
+		log.Error(err, logData)
 		setErrorCode(w, err)
 		return
 	}
 
 	setJSONContentType(w)
 	w.WriteHeader(http.StatusNoContent)
 
-	log.Info("got filtered blueprint", log.Data{"filter_blueprint_id": filterID, "dimension": name, "option": option})
+	log.Info("got filtered blueprint", logData)
 }
 
 func (api *FilterAPI) addFilterBlueprintDimensionOption(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
+	filterID := vars["filter_blueprint_id"]
+	name := vars["name"]
+	option := vars["option"]
+	logData := log.Data{
+		"func":                "addFilterBlueprintDimensionOption",
```

Comment:
I wonder if its necessary to have the func names in logs. Having the first log.info in the function provides the required context to determine the function. Also I can imagine them becoming stale if the function name changes.

Reply:
Good point, it was helpful whilst developing but maybe not so much in kibana